res/cpp/run_cpplint.py

```
@@ -0,0 +1,34 @@
+        for filename in filenames:
+            cl.ProcessFile(str(filename), cl_state.verbose_level)
+        cl_state.PrintErrorCounts()
+
+        error_count += cl_state.error_count
+        print('')
+
+    if error_count > 0:
+        raise RuntimeError("Codestyle check by cpplint failed")
+
+except ImportError:
+    warnings.warn("Stylecheck by cpplint failed because cpplint "
```

Ruff

Undefined name `warnings`

(F821)